spec/lcm/integration/shared_examples_for_synchronization_bricks.rb

```diff
@@ -132,9 +127,10 @@
   end
 
   it 'migrates metrics' do
-    used_in_dashboard = 'metric.customers.count'
-    tagged_production = Support::PRODUCTION_TAGGED_METRIC
-    expected_metrics = [used_in_dashboard, tagged_production]
+    used_reports = original_project.dashboards.map{|d|d.tabs.map{|t|t.items.map{|i|i}}}.flatten.map{|r|r.obj}.uniq{|r|r.identifier}
```

Comment:
Identical code found in shared_examples_for_synchronization_bricks.rb:114, shared_examples_for_synchronization_bricks.rb:130 (mass = 112)